modules/plugin/shapefile/src/main/java/org/geotools/data/shapefile/ShapefileSetManager.java

```diff
@@ -115,6 +115,7 @@ protected DbaseFileReader openDbfReader(boolean indexed) throws IOException {
                 }
             } catch (IOException e) {
                 // could happen if cpg file does not exist remotely
+                LOGGER.log(Level.FINE, "Error reading cpg file", e);
```

Comment:
We have several users setting logging to "VERBOSE" and then complaining there are "errors" in the logs just because they see stack traces. Could you reword the message to at least state this is likely not fatal and could actually even be expected? (same for the other logs added in the PR, at least for files that are not required by the shapefile spec).

Reply:
I updated the logging.